ABSTRACT.md

```diff
@@ -0,0 +1,11 @@
+# RNA SV Simulator
+
+In multicellular organisms, nearly every cell contains the same genome and thus the same genes. However, different cells show different gene expression underlying the different functions and tissues. Recent work in the cancer field, has shown that gene expression is more significant to disease outcomes than the genetic mutation [1].
+
+Structural variants of genetic information are primary drivers of many diseases [2], making them a prime target for bioinformatic tools, but new tools require test datasets. Datasets are frequently large, ambiguous and subject to restrictive sharing permissions. However, sufficient numbers of publicly available genomes has allowed for the establishment of simulation tools of SVs.
+
+Transcriptome sequencing is a much newer field, with a very low number of publicly available datasets — particularly in rare variants, significant to disease [3]. Here we present a tool that effectively replicates the expression profiles of rare and custom SVs across different tissue profiles to allow for the training and testing of bioinformatic tools.
```

Comment:
This is a bit misleading. We are simulating transcriptomes but currently we make no attempt to adjust the expression profile of downstream genes or anything. So I think It might be more accurate to say something like: Here we present a tool which injects SVs into a variety of different expression profiles to allow for the training and testing of bioinformatic tools. Simulating the transcriptome reads allows for a known ground truth so that we accurately quantify performance of tools not only in terms of sensitivity but also specificity.

Reply:
Agreed, your description is more fitting. Updated.